docs/cloud-init.yml

```diff
@@ -24,10 +32,28 @@ users:
 write_files:
   - path: /tmp/bhima-mysql-user.sql
     content: |
-      CREATE USER 'bhima'@'localhost' IDENTIFIED BY 'HISCongo2013';
+      CREATE USER 'bhima'@'localhost' IDENTIFIED BY 'HISCongo2020';
       GRANT ALL PRIVILEGES ON * . * TO 'bhima'@'localhost';
       FLUSH PRIVILEGES;
 
+  - path: /tmp/.my.cnf
+    content: |
+      [client]
+      user=root
+      password=PASSWORD
```

Comment:
Where does the `PASSWORD` get its value ?

Reply:
Good question.  It's hidden on [this line](https://github.com/IMA-WorldHealth/bhima/pull/4539/files#diff-aad866a9311fd8ed6d4c5e99baf24a1eR144).

The command is
```sh
sed -i "s/PASSWORD/$BH_DATABASE_PASSWORD/g" /tmp/.my.cnf
```

To my knowledge, we can't use variables in the `content` part of this script.